datadog_checks_dev/datadog_checks/dev/tooling/testing.py

```diff
@@ -215,3 +218,9 @@ def get_changed_checks():
     changed_files[:] = testable_files(changed_files)
 
     return {line.split('/')[0] for line in changed_files}
+
+
+def get_tox_env_python_version(env):
+    match = re.match(PYTHON_MAJOR_PATTERN, env)
+    if match:
+        return match.group(1)
```

Comment:
```suggestion
        return int(match.group(1))
```

To match the `int` type of `DEFAULT_PYTHON_VERSION = 2`

If ok for this change, this line might need to be update too then `if env_python_version and env_python_version != str(python):`